lib/inspec/reporters.rb

```diff
@@ -0,0 +1,30 @@
+require 'inspec/reporters/base'
+require 'inspec/reporters/cli'
+require 'inspec/reporters/json'
+require 'inspec/reporters/json_min'
+require 'inspec/reporters/junit'
+
+module Inspec::Reporters
+  def self.render(config)
+    case config[:name]
+    when 'cli'
+      reporter = Inspec::Reporters::CLI.new(config)
+    when 'json'
+      reporter = Inspec::Reporters::Json.new(config)
+    when 'json-min'
+      reporter = Inspec::Reporters::JsonMin.new(config)
+    when 'junit'
+      reporter = Inspec::Reporters::Junit.new(config)
+    end
```

Comment:
Is it possible for this method to be called with a reporter name string we're not expecting? Should we have a default `else` here that raises a `NotImplementedError`?